guides/mobile_app_guidance.md

```diff
@@ -0,0 +1,40 @@
+Mobile Application Guidance
+===========================
+
+Scope
+-----
+
+This guidance applies to mobile application in-house development and management.
+
+This guidance is intended to cover all forms of mobile application: from fully "off-line capable" progressive web applications, through cross-platform apps to fully native apps.
+
+This document should be read in conjunction with the Mobile Application Standards.
+
+Caveats
+-------
+
+The huge increase in the use of mobile devices means that mobile application development is now of critical interest to the DEFRA family, even more so given the amount of field-work being carried out by workers within the DEFRA family.
+
+This is a fast moving area involving multiple platforms and diverse technology stacks: this guidance is early and provisional.
+
+Extra Security Guidance
+-----------------------
+
+Please ensure that you have read the "Extra Security Considerations" of the Mobile Application Standards before reading this section.
+
+The Mobile Application Standards refer to various mechanisms for encrypting traffic going to and from the app over usually public networks: specifically HTTPS/Transport Layer Security, use of Virtual Private Networks or use of an in-built application security wrapper. The potential use of in-built application security wrappers requires some further guidance. This technique, often called "application wrapping", is a service commonly offered by Mobile Device Management (MDM) software suites. It involves using the MDM suite in some way to allow the MDM software to add an extra layer of security to your application bundle, such as encryption for data traffic or locally stored application data. While the use of this technique cannot be entirely ruled out, it is declining in popularity as it has many disadvantages. Application wrapping complicates deployment, creating an extra step and meaning that the wrapped app may not work properly outside the MDM eco-system. Even worse, it involves allowing the "application wrapping service" to make last minute changes to your application bundle, in ways that are largely outside of your control and, because they involve security, are likely to be deliberately obscure. It is even possible that the "application wrapping service" will itself be changing in ways unknown to us, often in response to continuously evolving security threats, so that an app wrapped at one moment may not be identical to one wrapped at the next, even if we have not ourselves made any changes to the app. Should the app wrapping itself cause any issues with the working of the app, those issues can therefore be extremely difficult to resolve. App wrapping should therefore only be adopted as a last resort.
```

Comment:
Suggest we keep this concise and factual. After the second sentence I suggest we say "Application wrapping should only be adopted as a last resort" and link to an existing post elsewhere that backs this up (if there is one).

Reply:
Run passed Mike P (the Technical Architect for the mobile area) and revised... not sure it's shorter though, but it does go into the alternatives now, and is possibly a bit more nuanced.